modules/special-rules/pages/psychic-beacon.adoc

```diff
@@ -1,17 +1,19 @@
 = Psychic Beacon
 
 `Detachment ability`
+footnote:[{e40k-diff}
+The {e40k} version of this ability is exceptionally cheap and powerful -- for only a small cost, it allows all Mycetic Swarm detachments in the army to arrive in turn 1, automatically.
+We've revised the ability so that it scales with the number of detachments that have the ability, and so that it improves rather than bypasses the odds on the Reserve Detachment Arrival table.
+This also means that the earliest a Mycetic Swarm detachment can arrive from Reserves is turn 2, just like other Reserves.
+]
```

Comment:
To close the loop on our thread https://github.com/GameBrains/er-core/pull/36/files#r1032948054, how about something like this:
 
```suggestion
footnote:[{e40k-diff}
The {e40k} version of this ability is exceptionally cheap and powerful -- for only a small cost, it allows all Mycetic Swarm detachments in the army to arrive in turn 1, automatically.
We've revised the ability so that it (a) scales with the number of detachments that have the ability, (b) grants you a re-roll rather than a bypass of the Reserve Detachment Arrival table, and (c) you can use the re-roll for any Tyranid detachment in Reserves, not just Mycetic Swarms.
Note that this also means that the earliest a Mycetic Swarm detachment can arrive from Reserves is turn 2, just like other Reserves.
]
```